blog/2025/keycloak-kubecon25-japan-announce.adoc

```diff
@@ -0,0 +1,40 @@
+:title: Register now for KubeCon Japan in June
+:date: 2025-03-24
+:publish: true
+:author: Alexander Schwartz
+:preview: keycloak-kubecon-tokyo-2025.png
+:summary: This year is the first time there is a KubeCon in Japan, and the Keycloak project is excited to be par of it! Register today to join us on June 16-17 2025 in Tokyo, Japan for this exciting event.
```

Comment:
```suggestion
:summary: This year is the first time there is a KubeCon in Japan, and the Keycloak project is excited to be part of it! Register today to join us on June 16-17 2025 in Tokyo, Japan for this exciting event.
```

Did you meant `part` rather than `par` ?